### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by Diadlo**

### CODE REVIEW

File: src/core/core.cpp [hunk 1]
```diff
@@ -656,9 +656,9 @@ void Core::changeGroupTitle(int groupId, const QString& title)
         emit groupTitleChanged(groupId, getUsername(), title);
 }
 
-void Core::sendFile(uint32_t friendId, QString Filename, QString FilePath, long long filesize)
+void Core::sendFile(uint32_t friendId, QString filename, QString FilePath, long long filesize)
```
Comment (sudden6) [inline src/core/core.cpp]: why different case for `filename` and `filepath` ?

File: src/core/core.h [hunk 2]
```diff
@@ -115,7 +115,7 @@ public slots:
      int sendAction(uint32_t friendId, const QString& action);
     void sendTyping(uint32_t friendId, bool typing);
 
-    void sendFile(uint32_t friendId, QString Filename, QString FilePath, long long filesize);
+    void sendFile(uint32_t friendId, QString filename, QString FilePath, long long filesize);
```
Comment (sudden6) [inline src/core/core.h]: same as above

File: src/core/corefile.cpp [hunk 3]
```diff
@@ -89,7 +88,7 @@ void CoreFile::sendAvatarFile(Core* core, uint32_t friendId, const QByteArray& d
     addFile(friendId, fileNum, file);
 }
 
-void CoreFile::sendFile(Core* core, uint32_t friendId, QString Filename, QString FilePath, long long filesize)
+void CoreFile::sendFile(Core* core, uint32_t friendId, QString Filename, QString filePath, long long filesize)
```
Comment (sudden6) [inline src/core/corefile.cpp]: you missed one^^

Comment (Diadlo) [inline src/core/corefile.cpp]: Fixed :[

**Comment by zetok** [comment]
PR needs to be rebased, since there are now some conflicts.

**Comment by Diadlo** [comment]
@zetok Rebased

**Pull request merged**